src/shell/Models/ShellExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NeoShell.Models
+{
+    class ShellExtensions : List<ShellExtension>
+    {
+        public bool TryFindCommand(string[] args, out ShellExtension? command)
+        {
+            foreach (string arg in args)
```

Comment:
seems odd that we are searching *all* the args instead of just the first one.

Reply:
I was thinking potential non-positional parameters is it possible that we have "neosh --json worknet ..." ?

Reply:
it feels like that might lead to argument collision. For example, If I created a "transfer" extension, how would shell differentiate between that extension and the NEP11 extension transfer command? (assuming both installed)